stock_eggs/source-engine/egg-counter--strike--global-offensive.json

```diff
@@ -34,12 +39,12 @@
         },
         {
             "name": "Steam Account Token",
-            "description": "The Steam Account Token required for the server to be displayed publicly.",
+            "description": "The Steam Account Token required for the server to be displayed publicly.\r\nhttps:\/\/steamcommunity.com\/dev\/managegameservers",
             "env_variable": "STEAM_ACC",
             "default_value": "",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|string|alpha_num|size:32"
+            "rules": "nullable|alpha_num|size:32"
```

Comment:
If you want your server to show as online this needs to be set.

Reply:
yea, I though it should be client preference so they can use their token if they want server to be online.